mixer/adapter/inventory.go

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:generate go run $GOPATH/src/istio.io/istio/mixer/tools/codegen/cmd/mixgeninventory/main.go -o inventory.gen.go -f inventory.yaml
```

Comment:
@douglas-reid in this particular case, I am going to nuke this file `inventory.go` soon. Any suggestion where can we put this `generate go run` step ? We will have to add an empty file ?